docs/metrics/introduction/joins.md

```diff
@@ -0,0 +1,227 @@
+
+Given these example queries:
+
+```
+#A: metric=CPU_User account=* | avg by account,metric
+#B: metric=CPU_Sys account=* | avg by account,metric
+#C: #A + #B
+```
+
+Query #A returns three result sets:
+
+```
+metric=CPU_User, account=dev
+metric=CPU_User, account=stage
+metric=CPU_User, account=prod
+```
+
+Query #B returns three result sets:
+
+```
+metric=CPU_Sys, account=dev
+metric=CPU_Sys, account=stage
+metric=CPU_Sys, account=prod
+```
+
+Query #C returns 9 result sets:
+
+```
+(metric=CPU_User, account=dev) + (metric=CPU_Sys, account=dev)
+(metric=CPU_User, account=dev) + (metric=CPU_Sys, account=stage)
+(metric=CPU_User, account=dev) + (metric=CPU_Sys, account=prod)
+(metric=CPU_User, account=stage) + (metric=CPU_Sys, account=dev)
+(metric=CPU_User, account=stage) + (metric=CPU_Sys, account=stage)
+(metric=CPU_User, account=stage) + (metric=CPU_Sys, account=prod)
+(metric=CPU_User, account=prod) + (metric=CPU_Sys, account=dev)
+(metric=CPU_User, account=prod) + (metric=CPU_Sys, account=stage)
+(metric=CPU_User, account=prod) + (metric=CPU_Sys, account=prod)
+```
+
+<img src={useBaseUrl('img/metrics/join-results-without-along.png')} alt="join-results-without-along.png"/>
+
+Including `along account` in the query ensures that only result sets in which the value of the `account` field match should be used for computation. This narrows down the result set list to the result sets expected.
+
+Queries with modified #C row:
+
+```
+#A: metric=CPU_User account=* | avg by account,metric
+#B: metric=CPU_Sys account=* | avg by account,metric
+#C: #A + #B along account
+```
+
+return same result sets for row #A and #B and only three results for row #C:
+
+```
+(metric=CPU_User, account=dev) + (metric=CPU_Sys, account=dev)
+(metric=CPU_User, account=stage) + (metric=CPU_Sys, account=stage)
+(metric=CPU_User, account=prod) + (metric=CPU_Sys, account=prod)
+```
+<img src={useBaseUrl('img/metrics/three-rows.png')} alt="three-rows.png"/>
+
+## Quantization and joined queries
+
+The results of a query with a join are calculated separately for every quantization bucket. This can produce unexpected results when time series with misaligned quantization buckets are joined.
+
+For example, consider a joined query that compares the `CPU_User` metric from two different hosts.
+
+```
+#A: metric=CPU_User _source=”HostMetrics”
+#B: metric=CPU_User _source=”JBM Host Metrics”
+#C: #A+#B/2
+```
+
+Doing a quick comparison of metrics and average value over a short period can yield can surprising results, as the screenshot below illustrates.
+
+<img src={useBaseUrl('img/metrics/odd-join-results.png')} alt="odd-join-results.png"/>
+
+:::note
+ query #C computed the two values of the average value.
```

Comment:
```suggestion
query `#C` computed the two values of the average value.
```

Reply:
@kimsauce, I format #C as code in query examples, like this:

#C: #A + #B along account

But not when referring to it as a label. So, let's not change it.